### PR TITLE
Correct imports and remove some langchain dependancies

### PR DESCRIPTION
**Pull request by aymeric-roucher**
# What does this PR do?

Thisixes missing pip installs at the beginning, and also removes unnecessary dependancies (LangChain) that were creating a bug after a breaking change.

 @MKhalusova

### CODE REVIEW

File: notebooks/en/rag_evaluation.ipynb [hunk 1]
```diff
@@ -37,47 +37,76 @@
   },
```
Comment (MKhalusova) [inline notebooks/en/rag_evaluation.ipynb:37]: <p>Can you please remove the output of this cell? We don't need it, and it doesn't render well</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/50/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Comment (MKhalusova) [inline notebooks/en/rag_evaluation.ipynb:37]: <p>Here, you can also remove the output</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/50/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

**Comment by MKhalusova** [comment]
Thanks for the updates! Can you remove a couple of outputs?

**Comment by aymeric-roucher** [comment]
Outputs removed @MKhalusova ✅

**Review by MKhalusova — APPROVED** [review]
Thanks!

**Pull request merged**